### ISSUE TITLE
Add test for deprecated fields

### ISSUE THREAD

**Issue by stubailo** [apollostack/eslint-plugin-graphql#14]
Would be good to know what the message looks like, and if it works well!

**Comment by stubailo** [comment]
<img width="737" alt="screenshot 2016-04-27 11 38 23" src="https://cloud.githubusercontent.com/assets/448783/14863925/939bb4a2-0c6c-11e6-93b0-cd58fe00e61e.png">

**Comment by stubailo** [comment]
Tweeted at Lee Byron to ask if there is an easy solution: https://twitter.com/stubailo/status/725396098584592384

**Comment by jnwng** [comment]
looks like we'd just need to integrate with https://github.com/graphql/graphql-js/issues/389

**Comment by sdcooke** [comment]
I think this is as simple as https://github.com/apollographql/eslint-plugin-graphql/compare/master...sdcooke:deprecated to find the fields. I'm not sure how you would want to integrate this properly (e.g. making it an optional rule) and I haven't got my head around this codebase properly yet so someone else is probably much better placed to implement it fully.

Modifying the plugin directly I've got output along the lines of:

"The field X.X is deprecated. [deprecation message from schema]"

**Comment by stubailo** [comment]
Yeah I think a custom validation rule in the options would be good, or perhaps a separate option like "validateDeprecatedFields: true"

**Comment by koddsson** [comment]
This has been done in #93!
